src/instructlab/sdg/generate_data.py

```diff
@@ -206,7 +206,8 @@ def _sdg_init(ctx, pipeline):
     pd = platformdirs.PlatformDirs(
         appname=os.path.join("instructlab", "sdg"), multipath=True
     )
-    for d in pd.iter_data_dirs():
+    pipelines_dir = os.path.join(pd, "pipelines")
```

Comment:
Did you want to do this inside the loop ?  (with d)

Reply:
Yep, thanks - dumb mistake :+1: